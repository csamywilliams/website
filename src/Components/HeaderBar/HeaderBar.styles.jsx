import styled from 'styled-components';
import media from '../../Utilities/MediaQueries';

export const Header = styled.nav`
    height: 12vh;
    min-height: 6vh;
    padding: .5em 1em .5em 0;
    display: -webkit-box;
    display: -moz-box;
    display: -ms-flexbox;
    display: -webkit-flex;
    display: flex;
    flex-wrap: nowrap;
    justify-content: space-between;
    border-bottom: 1px solid ${props => props.theme.navbar.border};
    align-items: center;
`;

export const Logo = styled.img`
    text-decoration: none;
    width: 40%;
    height: 100%;
`;

export const LogoLink = styled.a`
    display: inline-block;
    margin-top: 10px;
    margin-left: 20px;
    text-decoration: none;
`;

export const Menu = styled.ul`
    list-style-type: none;
    display: flex;
    flex-direction: row;
    justify-content: flex-end;

    li {
        text-align: center;
        margin: 0;
    }

    ${media.tablet`
        list-style-type: none;
        display: none;
    `}

    ${media.phone`
        list-style-type: none;
        display: none;
    `}
`;

export const MenuItem = styled.li`
    li {
        text-align: center;
        margin: 0 auto;
    }
`;

export const MenuLink = styled.a`
    text-decoration: none;
    display: block;
    padding: 1em;
    color: ${props => props.theme.link.color};
    flex: 0 0 auto; 

    :hover {
        border-bottom: 5px ${props => props.theme.link.hover} solid;
    }
`;

export const Hamburger = styled.div`
    position: absolute;
    top: .3em;
    right: 2em;
    cursor: pointer; 
    font-size: 1.5em;
    display: none;

    &:before {
        content: "";
        position: absolute;
        top: 0.25em;
        left: 0;
        width: 1em;
        height: 0.125em;
        border-top: 0.375em double ${props => props.theme.hamburger.lines};
        border-bottom: 0.125em solid ${props => props.theme.hamburger.lines};
    }

    ${media.tablet`
        display: unset;
    `}

    ${media.phone`
        display: unset;
    `}
`;
