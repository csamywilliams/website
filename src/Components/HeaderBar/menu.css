
@media (max-width: 768px) {
    .showMenu {
        display: block;
        margin: 0 auto;
    }
    
    .hideMenu {
        display: none;
    } 

    .showLogo {
        display: unset;
    }
    
    .hideLogo {
        display: none;
    } 
}