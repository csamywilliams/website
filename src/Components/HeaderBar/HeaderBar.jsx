import React, { Component } from 'react';
import {Header, LogoLink, Logo, Menu, MenuItem, Hamburger, MenuLink } from './HeaderBar.styles';
import  './menu.css';
import logo from './Logo.png';

class HeaderBar extends Component {

    constructor(props) {
        super(props);
        this.state = {active: false};

        this.toggleMenu = this.toggleMenu.bind(this);
    }

    toggleMenu() {
        this.setState(state => ({
            active: !state.active
        }));
    }

    render() {
        return (
            <Header>
                <LogoLink className={this.state.active ? 'hideLogo': 'showLogo'} href="/" >
                    <Logo src={logo} alt="Bees of Bryncoch logo"/>
                </LogoLink>
                <Menu className={this.state.active ? 'showMenu': 'hideMenu'}>
                    <MenuItem>
                        <MenuLink href="/about">ABOUT</MenuLink>
                    </MenuItem>
                    <MenuItem>
                        <MenuLink href="/blog">BLOG</MenuLink>
                    </MenuItem>
                    <MenuItem>
                        <MenuLink href="/gallery">GALLERY</MenuLink>
                    </MenuItem>
                    <MenuItem>
                        <MenuLink href="/review">REVIEWS</MenuLink>
                    </MenuItem>
                    <MenuItem>
                        <MenuLink href="/contact">CONTACT</MenuLink>
                    </MenuItem>
                </Menu>
                <Hamburger onClick={this.toggleMenu}>
                    
                </Hamburger>
            </Header>       
        )
    }
}

export default HeaderBar;