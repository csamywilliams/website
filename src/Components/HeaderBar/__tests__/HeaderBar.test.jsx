import React from 'react';
import ReactDOM from 'react-dom';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import { shallow } from 'enzyme';

import HeaderBar from '../HeaderBar';

Enzyme.configure({ adapter: new Adapter() });

describe('HeaderBar', () => {

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<HeaderBar />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders correctly', () => {
    const wrapper = shallow(<HeaderBar />);
    expect(wrapper).toMatchSnapshot();
  });


});
