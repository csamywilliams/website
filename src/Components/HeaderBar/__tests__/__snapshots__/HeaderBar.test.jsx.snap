// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderBar renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <HeaderBar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <ForwardRef
          className="logo"
          href="/"
        >
          <ForwardRef
            alt="Bees of Bryncoch logo"
            src="Logo.png"
          />
        </ForwardRef>,
        <ForwardRef>
          <li>
            <ForwardRef
              href="/about"
            >
              ABOUT
            </ForwardRef>
          </li>
          <li>
            <ForwardRef
              href="/blog"
            >
              BLOG
            </ForwardRef>
          </li>
          <li>
            <ForwardRef
              href="/gallery"
            >
              GALLERY
            </ForwardRef>
          </li>
          <li>
            <ForwardRef
              href="/review"
            >
              REVIEWS
            </ForwardRef>
          </li>
          <li>
            <ForwardRef
              href="/contact"
            >
              CONTACT
            </ForwardRef>
          </li>
        </ForwardRef>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <ForwardRef
            alt="Bees of Bryncoch logo"
            src="Logo.png"
          />,
          "className": "logo",
          "href": "/",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "alt": "Bees of Bryncoch logo",
            "src": "Logo.png",
          },
          "ref": null,
          "rendered": null,
          "type": Object {
            "$$typeof": Symbol(react.forward_ref),
            "attrs": Array [],
            "componentStyle": ComponentStyle {
              "componentId": "sc-bwzfXH",
              "isStatic": true,
              "lastClassName": "dcwtMI",
              "rules": Array [
                "
    width: 100%;
    height: 100%;
",
              ],
            },
            "displayName": "styled.img",
            "foldedComponentIds": Array [],
            "render": [Function],
            "styledComponentId": "sc-bwzfXH",
            "target": "img",
            "toString": [Function],
            "warnTooManyClasses": [Function],
            "withComponent": [Function],
          },
        },
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-htpNat",
            "isStatic": true,
            "lastClassName": "bXseJD",
            "rules": Array [
              "
    width: 8vw;
    height: 6vw;
    padding: 0 1em 0.5em 8em;
",
            ],
          },
          "displayName": "styled.a",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-htpNat",
          "target": "a",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        },
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <li>
              <ForwardRef
                href="/about"
              >
                ABOUT
              </ForwardRef>
            </li>,
            <li>
              <ForwardRef
                href="/blog"
              >
                BLOG
              </ForwardRef>
            </li>,
            <li>
              <ForwardRef
                href="/gallery"
              >
                GALLERY
              </ForwardRef>
            </li>,
            <li>
              <ForwardRef
                href="/review"
              >
                REVIEWS
              </ForwardRef>
            </li>,
            <li>
              <ForwardRef
                href="/contact"
              >
                CONTACT
              </ForwardRef>
            </li>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <ForwardRef
                href="/about"
              >
                ABOUT
              </ForwardRef>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": "ABOUT",
                "href": "/about",
              },
              "ref": null,
              "rendered": "ABOUT",
              "type": Object {
                "$$typeof": Symbol(react.forward_ref),
                "attrs": Array [],
                "componentStyle": ComponentStyle {
                  "componentId": "sc-ifAKCX",
                  "isStatic": true,
                  "lastClassName": "bUNfVr",
                  "rules": Array [
                    "
    text-decoration: none;
    display: block;
    padding: 1em;
    color: #2D1710;

    :hover {
        background: darken(deepskyblue, 2%);
    }

    @media all and (max-width: 600px) {
        text-align: center; 
        padding: 10px;
        border-top: 1px solid rgba(255,255,255,0.3); 
        border-bottom: 1px solid rgba(0,0,0,0.1); 
    }
",
                  ],
                },
                "displayName": "styled.a",
                "foldedComponentIds": Array [],
                "render": [Function],
                "styledComponentId": "sc-ifAKCX",
                "target": "a",
                "toString": [Function],
                "warnTooManyClasses": [Function],
                "withComponent": [Function],
              },
            },
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <ForwardRef
                href="/blog"
              >
                BLOG
              </ForwardRef>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": "BLOG",
                "href": "/blog",
              },
              "ref": null,
              "rendered": "BLOG",
              "type": Object {
                "$$typeof": Symbol(react.forward_ref),
                "attrs": Array [],
                "componentStyle": ComponentStyle {
                  "componentId": "sc-ifAKCX",
                  "isStatic": true,
                  "lastClassName": "bUNfVr",
                  "rules": Array [
                    "
    text-decoration: none;
    display: block;
    padding: 1em;
    color: #2D1710;

    :hover {
        background: darken(deepskyblue, 2%);
    }

    @media all and (max-width: 600px) {
        text-align: center; 
        padding: 10px;
        border-top: 1px solid rgba(255,255,255,0.3); 
        border-bottom: 1px solid rgba(0,0,0,0.1); 
    }
",
                  ],
                },
                "displayName": "styled.a",
                "foldedComponentIds": Array [],
                "render": [Function],
                "styledComponentId": "sc-ifAKCX",
                "target": "a",
                "toString": [Function],
                "warnTooManyClasses": [Function],
                "withComponent": [Function],
              },
            },
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <ForwardRef
                href="/gallery"
              >
                GALLERY
              </ForwardRef>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": "GALLERY",
                "href": "/gallery",
              },
              "ref": null,
              "rendered": "GALLERY",
              "type": Object {
                "$$typeof": Symbol(react.forward_ref),
                "attrs": Array [],
                "componentStyle": ComponentStyle {
                  "componentId": "sc-ifAKCX",
                  "isStatic": true,
                  "lastClassName": "bUNfVr",
                  "rules": Array [
                    "
    text-decoration: none;
    display: block;
    padding: 1em;
    color: #2D1710;

    :hover {
        background: darken(deepskyblue, 2%);
    }

    @media all and (max-width: 600px) {
        text-align: center; 
        padding: 10px;
        border-top: 1px solid rgba(255,255,255,0.3); 
        border-bottom: 1px solid rgba(0,0,0,0.1); 
    }
",
                  ],
                },
                "displayName": "styled.a",
                "foldedComponentIds": Array [],
                "render": [Function],
                "styledComponentId": "sc-ifAKCX",
                "target": "a",
                "toString": [Function],
                "warnTooManyClasses": [Function],
                "withComponent": [Function],
              },
            },
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <ForwardRef
                href="/review"
              >
                REVIEWS
              </ForwardRef>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": "REVIEWS",
                "href": "/review",
              },
              "ref": null,
              "rendered": "REVIEWS",
              "type": Object {
                "$$typeof": Symbol(react.forward_ref),
                "attrs": Array [],
                "componentStyle": ComponentStyle {
                  "componentId": "sc-ifAKCX",
                  "isStatic": true,
                  "lastClassName": "bUNfVr",
                  "rules": Array [
                    "
    text-decoration: none;
    display: block;
    padding: 1em;
    color: #2D1710;

    :hover {
        background: darken(deepskyblue, 2%);
    }

    @media all and (max-width: 600px) {
        text-align: center; 
        padding: 10px;
        border-top: 1px solid rgba(255,255,255,0.3); 
        border-bottom: 1px solid rgba(0,0,0,0.1); 
    }
",
                  ],
                },
                "displayName": "styled.a",
                "foldedComponentIds": Array [],
                "render": [Function],
                "styledComponentId": "sc-ifAKCX",
                "target": "a",
                "toString": [Function],
                "warnTooManyClasses": [Function],
                "withComponent": [Function],
              },
            },
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <ForwardRef
                href="/contact"
              >
                CONTACT
              </ForwardRef>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": "CONTACT",
                "href": "/contact",
              },
              "ref": null,
              "rendered": "CONTACT",
              "type": Object {
                "$$typeof": Symbol(react.forward_ref),
                "attrs": Array [],
                "componentStyle": ComponentStyle {
                  "componentId": "sc-ifAKCX",
                  "isStatic": true,
                  "lastClassName": "bUNfVr",
                  "rules": Array [
                    "
    text-decoration: none;
    display: block;
    padding: 1em;
    color: #2D1710;

    :hover {
        background: darken(deepskyblue, 2%);
    }

    @media all and (max-width: 600px) {
        text-align: center; 
        padding: 10px;
        border-top: 1px solid rgba(255,255,255,0.3); 
        border-bottom: 1px solid rgba(0,0,0,0.1); 
    }
",
                  ],
                },
                "displayName": "styled.a",
                "foldedComponentIds": Array [],
                "render": [Function],
                "styledComponentId": "sc-ifAKCX",
                "target": "a",
                "toString": [Function],
                "warnTooManyClasses": [Function],
                "withComponent": [Function],
              },
            },
            "type": "li",
          },
        ],
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "attrs": Array [],
          "componentStyle": ComponentStyle {
            "componentId": "sc-bxivhb",
            "isStatic": true,
            "lastClassName": "jLjcRm",
            "rules": Array [
              "
    list-style: none;
    margin: 0; 
    display: -webkit-box;
    display: -moz-box;
    display: -ms-flexbox;
    display: -webkit-flex;
    display: flex;
    -webkit-flex-flow: row wrap;
    justify-content: flex-end;
    align-items: center;
    padding-right: 5%;

    @media all and (max-width: 800px) {
        justify-content: space-around;
    }

    @media all and (max-width: 600px) {
        -webkit-flex-flow: column wrap;
        flex-flow: column wrap;
        padding: 0;
    }

    li {
        width: 5vw;
    }
",
            ],
          },
          "displayName": "styled.ul",
          "foldedComponentIds": Array [],
          "render": [Function],
          "styledComponentId": "sc-bxivhb",
          "target": "ul",
          "toString": [Function],
          "warnTooManyClasses": [Function],
          "withComponent": [Function],
        },
      },
    ],
    "type": Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "lastClassName": "fZPfvV",
        "rules": Array [
          "
    height: 12vh;
    min-height: 6vh;
    padding: .5em 1em .5em 0;
    display: -webkit-box;
    display: -moz-box;
    display: -ms-flexbox;
    display: -webkit-flex;
    display: flex;
    justify-content: space-between;
",
        ],
      },
      "displayName": "styled.header",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "header",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <ForwardRef
            className="logo"
            href="/"
          >
            <ForwardRef
              alt="Bees of Bryncoch logo"
              src="Logo.png"
            />
          </ForwardRef>,
          <ForwardRef>
            <li>
              <ForwardRef
                href="/about"
              >
                ABOUT
              </ForwardRef>
            </li>
            <li>
              <ForwardRef
                href="/blog"
              >
                BLOG
              </ForwardRef>
            </li>
            <li>
              <ForwardRef
                href="/gallery"
              >
                GALLERY
              </ForwardRef>
            </li>
            <li>
              <ForwardRef
                href="/review"
              >
                REVIEWS
              </ForwardRef>
            </li>
            <li>
              <ForwardRef
                href="/contact"
              >
                CONTACT
              </ForwardRef>
            </li>
          </ForwardRef>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <ForwardRef
              alt="Bees of Bryncoch logo"
              src="Logo.png"
            />,
            "className": "logo",
            "href": "/",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "alt": "Bees of Bryncoch logo",
              "src": "Logo.png",
            },
            "ref": null,
            "rendered": null,
            "type": Object {
              "$$typeof": Symbol(react.forward_ref),
              "attrs": Array [],
              "componentStyle": ComponentStyle {
                "componentId": "sc-bwzfXH",
                "isStatic": true,
                "lastClassName": "dcwtMI",
                "rules": Array [
                  "
    width: 100%;
    height: 100%;
",
                ],
              },
              "displayName": "styled.img",
              "foldedComponentIds": Array [],
              "render": [Function],
              "styledComponentId": "sc-bwzfXH",
              "target": "img",
              "toString": [Function],
              "warnTooManyClasses": [Function],
              "withComponent": [Function],
            },
          },
          "type": Object {
            "$$typeof": Symbol(react.forward_ref),
            "attrs": Array [],
            "componentStyle": ComponentStyle {
              "componentId": "sc-htpNat",
              "isStatic": true,
              "lastClassName": "bXseJD",
              "rules": Array [
                "
    width: 8vw;
    height: 6vw;
    padding: 0 1em 0.5em 8em;
",
              ],
            },
            "displayName": "styled.a",
            "foldedComponentIds": Array [],
            "render": [Function],
            "styledComponentId": "sc-htpNat",
            "target": "a",
            "toString": [Function],
            "warnTooManyClasses": [Function],
            "withComponent": [Function],
          },
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <li>
                <ForwardRef
                  href="/about"
                >
                  ABOUT
                </ForwardRef>
              </li>,
              <li>
                <ForwardRef
                  href="/blog"
                >
                  BLOG
                </ForwardRef>
              </li>,
              <li>
                <ForwardRef
                  href="/gallery"
                >
                  GALLERY
                </ForwardRef>
              </li>,
              <li>
                <ForwardRef
                  href="/review"
                >
                  REVIEWS
                </ForwardRef>
              </li>,
              <li>
                <ForwardRef
                  href="/contact"
                >
                  CONTACT
                </ForwardRef>
              </li>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <ForwardRef
                  href="/about"
                >
                  ABOUT
                </ForwardRef>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": "ABOUT",
                  "href": "/about",
                },
                "ref": null,
                "rendered": "ABOUT",
                "type": Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "attrs": Array [],
                  "componentStyle": ComponentStyle {
                    "componentId": "sc-ifAKCX",
                    "isStatic": true,
                    "lastClassName": "bUNfVr",
                    "rules": Array [
                      "
    text-decoration: none;
    display: block;
    padding: 1em;
    color: #2D1710;

    :hover {
        background: darken(deepskyblue, 2%);
    }

    @media all and (max-width: 600px) {
        text-align: center; 
        padding: 10px;
        border-top: 1px solid rgba(255,255,255,0.3); 
        border-bottom: 1px solid rgba(0,0,0,0.1); 
    }
",
                    ],
                  },
                  "displayName": "styled.a",
                  "foldedComponentIds": Array [],
                  "render": [Function],
                  "styledComponentId": "sc-ifAKCX",
                  "target": "a",
                  "toString": [Function],
                  "warnTooManyClasses": [Function],
                  "withComponent": [Function],
                },
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <ForwardRef
                  href="/blog"
                >
                  BLOG
                </ForwardRef>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": "BLOG",
                  "href": "/blog",
                },
                "ref": null,
                "rendered": "BLOG",
                "type": Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "attrs": Array [],
                  "componentStyle": ComponentStyle {
                    "componentId": "sc-ifAKCX",
                    "isStatic": true,
                    "lastClassName": "bUNfVr",
                    "rules": Array [
                      "
    text-decoration: none;
    display: block;
    padding: 1em;
    color: #2D1710;

    :hover {
        background: darken(deepskyblue, 2%);
    }

    @media all and (max-width: 600px) {
        text-align: center; 
        padding: 10px;
        border-top: 1px solid rgba(255,255,255,0.3); 
        border-bottom: 1px solid rgba(0,0,0,0.1); 
    }
",
                    ],
                  },
                  "displayName": "styled.a",
                  "foldedComponentIds": Array [],
                  "render": [Function],
                  "styledComponentId": "sc-ifAKCX",
                  "target": "a",
                  "toString": [Function],
                  "warnTooManyClasses": [Function],
                  "withComponent": [Function],
                },
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <ForwardRef
                  href="/gallery"
                >
                  GALLERY
                </ForwardRef>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": "GALLERY",
                  "href": "/gallery",
                },
                "ref": null,
                "rendered": "GALLERY",
                "type": Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "attrs": Array [],
                  "componentStyle": ComponentStyle {
                    "componentId": "sc-ifAKCX",
                    "isStatic": true,
                    "lastClassName": "bUNfVr",
                    "rules": Array [
                      "
    text-decoration: none;
    display: block;
    padding: 1em;
    color: #2D1710;

    :hover {
        background: darken(deepskyblue, 2%);
    }

    @media all and (max-width: 600px) {
        text-align: center; 
        padding: 10px;
        border-top: 1px solid rgba(255,255,255,0.3); 
        border-bottom: 1px solid rgba(0,0,0,0.1); 
    }
",
                    ],
                  },
                  "displayName": "styled.a",
                  "foldedComponentIds": Array [],
                  "render": [Function],
                  "styledComponentId": "sc-ifAKCX",
                  "target": "a",
                  "toString": [Function],
                  "warnTooManyClasses": [Function],
                  "withComponent": [Function],
                },
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <ForwardRef
                  href="/review"
                >
                  REVIEWS
                </ForwardRef>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": "REVIEWS",
                  "href": "/review",
                },
                "ref": null,
                "rendered": "REVIEWS",
                "type": Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "attrs": Array [],
                  "componentStyle": ComponentStyle {
                    "componentId": "sc-ifAKCX",
                    "isStatic": true,
                    "lastClassName": "bUNfVr",
                    "rules": Array [
                      "
    text-decoration: none;
    display: block;
    padding: 1em;
    color: #2D1710;

    :hover {
        background: darken(deepskyblue, 2%);
    }

    @media all and (max-width: 600px) {
        text-align: center; 
        padding: 10px;
        border-top: 1px solid rgba(255,255,255,0.3); 
        border-bottom: 1px solid rgba(0,0,0,0.1); 
    }
",
                    ],
                  },
                  "displayName": "styled.a",
                  "foldedComponentIds": Array [],
                  "render": [Function],
                  "styledComponentId": "sc-ifAKCX",
                  "target": "a",
                  "toString": [Function],
                  "warnTooManyClasses": [Function],
                  "withComponent": [Function],
                },
              },
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <ForwardRef
                  href="/contact"
                >
                  CONTACT
                </ForwardRef>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": "CONTACT",
                  "href": "/contact",
                },
                "ref": null,
                "rendered": "CONTACT",
                "type": Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "attrs": Array [],
                  "componentStyle": ComponentStyle {
                    "componentId": "sc-ifAKCX",
                    "isStatic": true,
                    "lastClassName": "bUNfVr",
                    "rules": Array [
                      "
    text-decoration: none;
    display: block;
    padding: 1em;
    color: #2D1710;

    :hover {
        background: darken(deepskyblue, 2%);
    }

    @media all and (max-width: 600px) {
        text-align: center; 
        padding: 10px;
        border-top: 1px solid rgba(255,255,255,0.3); 
        border-bottom: 1px solid rgba(0,0,0,0.1); 
    }
",
                    ],
                  },
                  "displayName": "styled.a",
                  "foldedComponentIds": Array [],
                  "render": [Function],
                  "styledComponentId": "sc-ifAKCX",
                  "target": "a",
                  "toString": [Function],
                  "warnTooManyClasses": [Function],
                  "withComponent": [Function],
                },
              },
              "type": "li",
            },
          ],
          "type": Object {
            "$$typeof": Symbol(react.forward_ref),
            "attrs": Array [],
            "componentStyle": ComponentStyle {
              "componentId": "sc-bxivhb",
              "isStatic": true,
              "lastClassName": "jLjcRm",
              "rules": Array [
                "
    list-style: none;
    margin: 0; 
    display: -webkit-box;
    display: -moz-box;
    display: -ms-flexbox;
    display: -webkit-flex;
    display: flex;
    -webkit-flex-flow: row wrap;
    justify-content: flex-end;
    align-items: center;
    padding-right: 5%;

    @media all and (max-width: 800px) {
        justify-content: space-around;
    }

    @media all and (max-width: 600px) {
        -webkit-flex-flow: column wrap;
        flex-flow: column wrap;
        padding: 0;
    }

    li {
        width: 5vw;
    }
",
              ],
            },
            "displayName": "styled.ul",
            "foldedComponentIds": Array [],
            "render": [Function],
            "styledComponentId": "sc-bxivhb",
            "target": "ul",
            "toString": [Function],
            "warnTooManyClasses": [Function],
            "withComponent": [Function],
          },
        },
      ],
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "lastClassName": "fZPfvV",
          "rules": Array [
            "
    height: 12vh;
    min-height: 6vh;
    padding: .5em 1em .5em 0;
    display: -webkit-box;
    display: -moz-box;
    display: -ms-flexbox;
    display: -webkit-flex;
    display: flex;
    justify-content: space-between;
",
          ],
        },
        "displayName": "styled.header",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "header",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
